Classes/Private/Manager/Configuration/OCTManagerConfiguration.m

```diff
@@ -43,6 +43,7 @@ + (instancetype)defaultConfiguration
     configuration.options.proxyType = OCTToxProxyTypeNone;
     configuration.options.proxyHost = nil;
     configuration.options.proxyPort = 0;
+    configuration.options.tcpPort = 0;
```

Comment:
I did a git revert on one of the previous when we were changing tox to mannol's branch. I think this is where it came from?